ai_ta_backend/database/sql.py

```diff
@@ -136,4 +136,7 @@ def getDisabledDocGroups(self, course_name: str):
     return self.supabase_client.table("doc_groups").select("name").eq("course_name", course_name).eq("enabled", False).execute()
 
   def insertProject(self, project_info):
-    return self.supabase_client.table("projects").insert(project_info).execute()
\ No newline at end of file
+    return self.supabase_client.table("projects").insert(project_info).execute()
+  
+  def getPreAssignedAPIKeys(self, email: str):
+    return self.supabase_client.table("pre_authorized_api_keys").select("*").contains("emails", '["' + email + '"]').execute()
```

Comment:
is this the most reliable way to do this? The syntax seems weird, but if it works it works. 

Is this useful `json.loads( ... )` and `json.dump()`? I'm 100% not sure, I haven't checked yet. I'll dig in tomorrow.